mitodl/mitxpro
# Delete more obsolete github templates

In https://github.com/mitodl/mitxpro/pull/2843 I removed the PULL_REQUEST_TEMPLATE directory, but I overlooked another pull request template. 

While I'm at it, I'm going to remove the issue templates as well. All these should be coming from the central mitodl organization's templates. 

Delete .github/ISSUE_TEMPLATE directory

#### What are the relevant tickets?

none 

#### What's this PR do?

no more custom templates

#### How should this be manually tested?

It can't be tested until it's merged.

After it's merged, try opening a PR or an issue and see if it uses the standard template defined at https://github.com/mitodl/.github/tree/main/.github

#### Where should the reviewer start?

Did I get all the templates this time?
